code/modules/clothing/spacesuits/rig/suits/combat.dm

```diff
@@ -26,4 +25,59 @@
 		/obj/item/rig_module/power_sink,
 		/obj/item/rig_module/electrowarfare_suite,
 		/obj/item/rig_module/chem_dispenser/combat
+		)
+
+/obj/item/weapon/rig/combat/empty
+	initial_modules = list(
+		/obj/item/rig_module/ai_container,
+		/obj/item/rig_module/electrowarfare_suite,
+		)
+
+/obj/item/weapon/rig/military
+	name = "military hardsuit control module"
+	desc = "An austere hardsuit used by paramilitary groups and real soldiers alike."
+	icon_state = "military_rig"
+	suit_type = "military hardsuit"
+	armor = list(melee = 80, bullet = 70, laser = 55, energy = 15, bomb = 80, bio = 100, rad = 30)
+	slowdown = 1
+	offline_slowdown = 3
+	offline_vision_restriction = 1
+	allowed = list(/obj/item/device/flashlight, /obj/item/weapon/tank,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/weapon/handcuffs,/obj/item/device/t_scanner, /obj/item/weapon/rcd, /obj/item/weapon/tool/crowbar, \
```

Comment:
If every type of `obj/item/weapon/tool` can fit in this, you can just have that path in the allowed, rather than using all 4 of them.